scripts/shared/clusters.sh

```diff
@@ -114,7 +119,7 @@ mkdir -p ${KUBECONFIGS_DIR}
 
 run_local_registry
 declare_cidrs
-with_retries 3 run_parallel "{1..3}" create_kind_cluster
+with_retries 3 run_all_clusters create_kind_cluster
 declare_kubeconfig
-run_parallel "2 3" deploy_weave_cni
+run_subm_clusters deploy_weave_cni
```

Comment:
This doesn't look quite equivalent to prior behavior for submariner. `run_subm_clusters` will run it on any cluster where `cluster_subm[X]` is true. Prior behavior hard-coded to omit `cluster1`. However cluster_settings has `cluster_subm['cluster1']` set to true so now weave will be deployed there.

Reply:
Oh I see you ignore "cluster1" in `deploy_weave_cni`.